en/simulation/flightgear.md

```diff
@@ -132,7 +132,7 @@ INFO  [mavlink] mode: Onboard, data rate: 4000 B/s on udp port 14280 remote port
 INFO  [logger] logger started (mode=all)
 INFO  [logger] Start file log (type: full)
 INFO  [logger] Opened full log file: ./log/2020-04-28/22_03_36.ulg
-INFO  [mavlink] MAVLink only on localhost (set param MAV_BROADCAST = 1 to enable network)
+INFO  [mavlink] MAVLink only on localhost
```

Comment:
Any idea what you would see here?

Reply:
```
INFO  [mavlink] MAVLink only on localhost (set param MAV_{i}_BROADCAST = 1 to enable network)
```
Which isn't correct though for SITL.

Reply:
@bkueng I didn't ignore you last time you mentioned MAV_x_BROADCAST. I just assumed you were wrong because I can't find this parameter in http://docs.px4.io/master/en/advanced_config/parameter_reference.html .
What's going on?

Reply:
Ah, looking below I think you're saying this will only appear for a board that supports it - Ethernet etc. Hmmm. So not linkable in docs since not generated.

Reply:
Correct. Indeed, it's getting more difficult the more board-specific generation we have.
For this case we can pretend there's ethernet, so the generator adds the params if you want.

Reply:
@bkueng Yes please - can you make the generator changes needed.